bolt-modules/boltlib/lib/puppet/functions/wait.rb

```diff
@@ -0,0 +1,75 @@
+# frozen_string_literal: true
+
+require 'bolt/logger'
+require 'bolt/target'
+
+# Wait for a Future or array of Futures to finish and return results.
+#
+# > **Note:** Not available in apply block
+Puppet::Functions.create_function(:wait, Puppet::Functions::InternalFunction) do
+  # Wait for a Future or array of Futures to finish and return results.
```

Comment:
We should have a brief summary of this specific dispatch instead of a description of the function, as this is used as the header for the function signature. Maybe something like "Wait for Futures to finish" ?

See [run_command](https://github.com/puppetlabs/bolt/blob/main/bolt-modules/boltlib/lib/puppet/functions/run_command.rb#L6-L11) as an example.

Reply:
Oof I just totally forgot to do this, thanks!